ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/results/clientlibs/site/less/excerpt.less

```diff
@@ -0,0 +1,24 @@
+.excerpt {
+    padding: 20px;
+    box-sizing: border-box;
+    letter-spacing: 0.1px;
+}
+
+.excerpt:first-letter {
+    line-height: 24px;
+    margin: 0 5px 0 0;
+    padding: 7px 0 3px 0;
+    font-size: 40px;
+    float: left;
+    text-transform: capitalize;
+    display: inline-block;
+}
+.excerpt-elements {
+    padding: 15px;
+    box-sizing: border-box;
+    letter-spacing: 0.1px;
+    	hr {
+    		border: none;
+          	border-top: .0625rem solid #e6e6e6;
+        }
+}
```

Comment:
Can the :hover color on the excerpt be removed?
![image](https://user-images.githubusercontent.com/8974514/40268334-7e0e3aec-5b39-11e8-9e5a-d998d23d8c4b.png)